bindings/python/python/rspdl/__init__.py

```
@@ -0,0 +1,115 @@
+    result: Any
+
+
+def _decode(response_json: str) -> SdkResponse:
+    return json.loads(response_json)
+
+
+def _encode(request: Mapping[str, Any]) -> str:
+    return json.dumps(request, ensure_ascii=False, separators=(",", ":"))
+
+
+def compile(
```

Ruff (0.16.1)

[error] Variable `compile` is shadowing a Python builtin

(A001)